tests/shopify_python/test_git_utils.py

```diff
@@ -248,3 +248,67 @@ def test_autopep_files(tmpdir):
     for fixed_file in files_to_autopep:
         with open(fixed_file) as file_to_check:
             assert file_to_check.readlines() == file_lines
+
+
+def test_linter(tmpdir):
+    # type: ('py.path.LocalPath') -> None
+    open(str(tmpdir.join('__init__.py')), 'w')
+
+    file_lines = [
+        "def foo():",
+        "  return 1\n"
+    ]
+    tmpdir.join('file.py').write('\n'.join(file_lines))
```

Comment:
> The only way to use `"""` syntax would be to left-align all of the text flush with the left margin of the page.

[The pylint linter tests](https://github.com/Shopify/shopify_python/blob/master/tests/shopify_python/test_google_styleguide.py#L179) get away with it b/c the parser function appears to be ignoring the initial whitespace. You could implement a function wrapper to accomplish the same thing so that you don't need to left-align to the margin.